esrally/mechanic/launcher.py

```diff
@@ -348,10 +348,9 @@ def _set_env(self, env, k, v, separator=' ', prepend=False):
         if v is not None:
             if k not in env:
                 env[k] = v
-            else:  # merge
-                if (prepend == True):
+            elif (prepend == True):
```

Comment:
While this will work correctly, idiomatic usage is `elif prepend` instead of `elif (prepend == True)`. Can you please change that?